src/04-modality-specific-files/05-task-events.md

```diff
@@ -65,6 +65,28 @@ onset duration  trial_type  response_time stim_file
 5.6 0.6 stop  1.739 images/blue_square.jpg
 ```
 
+The accompanying JSON side car might be:
+
+```Text
+sub-control01/
+    func/
+        sub-control01_task-stopsignal_events.json
+```
+
+```JSON
+{
+   "trial_type": {
+       "LongName": "Event category",
+       "Description": "Indicator of type of action that is expected",
+       "Levels": {
+          "go": "A red square is displayed to indicate starting",
+          "stop": "A blue square is displayed to indicate stopping",
+       }
+    }
+}
+```	   
```

Comment:
Thanks for adding the detailed JSON example.

I am just not sure whether or not that JSON example should be *complete* or not.

Currently, it's incomplete: It only describes the `trial_type` column

To make it complete, we'd also have to either:
1.  describe ALL columns including `onset`, `duration`, `trial_type`, `response_time`, `stim_file`
1. or describe all NON-STANDARD columns ... then we could drop `onset`, `duration`, and `stim_file` from the example JSON, because these three columns are fully defined in the BIDS specification

I am tending towards describing all columns (my option 1) --> although that'll mean a long example. :man_shrugging:

Reply:
I would like to separate the issue of documenting the columns in a different discussion and a different pull request. Currently the events.json doesn't have a schema in the bids-validator (issue #1031), so the validation is limited. The meaning of onset and duration are fairly specifically described in the spec and are required.